src/pages/workspace/companyCards/WorkspaceCompanyCardsPage.tsx

```diff
@@ -125,88 +116,71 @@ type WorkspaceCompanyCardPageProps = StackScreenProps<FullScreenNavigatorParamLi
 
 function WorkspaceCompanyCardPage({route}: WorkspaceCompanyCardPageProps) {
     const {translate} = useLocalize();
+    const styles = useThemeStyles();
+    const theme = useTheme();
     const policyID = route.params.policyID;
-    // const workspaceAccountID = PolicyUtils.getWorkspaceAccountID(policyID);
-
-    // TODO: use data form onyx instead of mocked one when API is implemented
-    // const [cardFeeds] = useOnyx(`${ONYXKEYS.COLLECTION.SHARED_NVP_PRIVATE_DOMAIN_MEMBER}${workspaceAccountID}`);
-    const cardFeeds = mockedFeeds;
+    const workspaceAccountID = PolicyUtils.getWorkspaceAccountID(policyID);
+    const [cardFeeds] = useOnyx(`${ONYXKEYS.COLLECTION.SHARED_NVP_PRIVATE_DOMAIN_MEMBER}${workspaceAccountID}`);
     const [lastSelectedFeed] = useOnyx(`${ONYXKEYS.COLLECTION.LAST_SELECTED_FEED}${policyID}`);
     const defaultFeed = Object.keys(cardFeeds?.companyCards ?? {})[0];
     const selectedFeed = lastSelectedFeed ?? defaultFeed;
+    const fetchCompanyCards = useCallback(() => {
+        Policy.openPolicyCompanyCardsPage(policyID, workspaceAccountID);
+    }, [policyID, workspaceAccountID]);
+
+    useFocusEffect(fetchCompanyCards);
```

Comment:
@waterim do we really need to call this api on focus, shouldn't it only call on the mount of the component? Bcoz I think other onyx data will be updated with the WRITE api responses.